### PR TITLE
Javadoc: Update links and fix typos

### PR DESCRIPTION
**Pull request by apjanke**
Updates various outdated/broken links, and fixes some typos and grammatical errors.

Changes are restricted to the Javadocs.

### CODE REVIEW

File: src/main/java/com/google/maps/DirectionsApiRequest.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public DirectionsApiRequest destination(String destination) {
   }
 
   /**
-   * The origin, as a latitude,longitude location.
+   * The origin, as a latitude/longitude location.
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: /cc @sarahmaddox I'm unsure about this change, thoughts?

Comment (sarahmaddox) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: LGTM

File: src/main/java/com/google/maps/GeocodingApiRequest.java [hunk 2]
```diff
@@ -86,11 +86,11 @@ public GeocodingApiRequest latlng(LatLng latlng) {
 
   /**
    * Set the bounding box of the viewport within which to bias geocode results more prominently.
-   * This parameter will only influence, not fully restrict, results from the geocoder. (
+   * This parameter will only influence, not fully restrict, results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#Viewports">Viewports
-   * documentation</a>.
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#Viewports">
```
Comment (sarahmaddox) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: While you're here, please would you fix this URL? It should be:
https://developers.google.com/maps/documentation/geocoding/start#Viewports

FYI, the URL parameter "hl=pl" shouldn't be there. It's requesting a version of the page in Polish (which we don't actually have) and is probably the result of someone from Poland making this change while viewing the docs.

To see the language specification in action, try this URL which requests the page in German:
https://developers.google.com/maps/documentation/geocoding/start?hl=de

If you leave the parameter off the URL, the developer docs site will look for language info in the user's browser or as specified in the language option at the bottom of the page.

File: src/main/java/com/google/maps/GeocodingApiRequest.java [hunk 3]
```diff
@@ -105,8 +105,8 @@ public GeocodingApiRequest bounds(LatLng southWestBound, LatLng northEastBound)
    * parameter will only influence, not fully restrict, results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#RegionCodes">Region
-   * Codes</a>.
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#RegionCodes">Region
```
Comment (sarahmaddox) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: Similar to my previous comment, the URL should be:
https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes

File: src/main/java/com/google/maps/GeocodingApiRequest.java [hunk 4]
```diff
@@ -120,7 +120,7 @@ public GeocodingApiRequest region(String region) {
    * fully restrict the results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#ComponentFiltering">
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#ComponentFiltering">
```
Comment (sarahmaddox) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: Similar to my previous comment, the URL should be:
https://developers.google.com/maps/documentation/geocoding/intro#ComponentFiltering

File: src/main/java/com/google/maps/DirectionsApiRequest.java (hunk 1 above)
Comment (apjanke) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: I browsed through the whole source tree, and "latitude/longitude" seems to be a lot more common that "latitude,longitude". Seems like it should have parens and a space if it's going to be "(latitude, longitude)".

File: src/main/java/com/google/maps/GeocodingApiRequest.java (hunk 2 above)
Comment (apjanke) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: Fixed.

File: src/main/java/com/google/maps/GeocodingApiRequest.java (hunk 3 above)
Comment (apjanke) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: Fixed.

File: src/main/java/com/google/maps/GeocodingApiRequest.java (hunk 4 above)
Comment (apjanke) [inline src/main/java/com/google/maps/GeocodingApiRequest.java]: Fixed.

**Comment by apjanke** [comment]
Fixed the `hl=pl` things. Sorry for not noticing that earlier.

File: src/main/java/com/google/maps/DirectionsApiRequest.java (hunk 1 above)
Comment (domesticmouse) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: sgtm

**Pull request merged**